Monika After Story/game/script-grammar.rpy

```diff
@@ -0,0 +1,396 @@
+    m 1hua "Ehehe."
+    m 1eub "Okay, that's enough for today, [player]."
+    m 3hub "Thanks for listening!"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_gtod_tip005",
+            category=["grammar tips"],
+            prompt="Subjects and Objects",
+            conditional="store.mas_gtod.has_day_past_tip(4)",
+            action=EV_ACT_POOL
+        )
+    )
+
+label monika_gtod_tip005:
+    m 1eua "Today we'll talk about subjects and objects, [player]."
+    m 1eud "Remember when I told you about clauses having an action and a verb?"
+    m 3eub "The object is the person or thing that the subject acts on!"
+    m 1eua "So, in the sentence '{b}We watched the fireworks together{/b},' the object would be...{w=0.5}the '{b}fireworks{/b}.'"
+    m 3esd "Oh, it's important to note that objects aren't necessary to form complete sentences..."
+    m 1eua "The sentence could very well have been, '{b}We watched.{/b}'"
+    m 3hksdlb "That's a complete sentence...although it's an ambiguous one, ahaha!"
+    m 1eud "There's also nothing that says that the object has to come last, but I'll discuss that in more detail another time."
+    m 3esa "Just remember that the subject is doing the action and the object is acted upon."
+    m 1eub "Okay, that's all for today..."
+    m 3hub "Thanks for listening, [player]! I love."
+    m 1eua "..."
+    m 1tuu "..."
+    m 3hub "You!"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_gtod_tip006",
+            category=["grammar tips"],
+            prompt="Active and Passive Voices",
+            conditional="store.mas_gtod.has_day_past_tip(5)",
+            action=EV_ACT_POOL
+        )
+    )
+
+label monika_gtod_tip006:
+    m 1eud "[player], do you know about voices in writing?"
+    m 3eua "There's the active voice and the passive voice."
+    m 3euc "If you remember our talk on subjects and objects, the big difference between the two voices is whether the subject or the object comes first."
+    m 1esd "Let's say the subject is '{b}Sayori{/b}' and the object is a '{b}cupcake{/b}.'"
+    m 3eud "Here's the sentence in an active voice: {w=0.5}'{b}Sayori ate the last cupcake.{/b}'"
+    m 3euc "Here it is again in a passive voice: {w=0.5}'{b}The last cupcake was eaten.{/b}'"
+    m 1eub "As you can see, you can use the passive voice to be secretive about the subject yet still have a complete sentence."
+    m 1tuu "You {i}can{/i} use the passive voice to be sneaky! {w=0.5}It does have other uses too, though."
+    m 3esd "For example, in some careers, people have to use the passive voice to be impersonal."
+    m 3euc "Scientists describe experiments with '{b}the data was taken{/b}...' since the important part is their work and not who did it."
```

Comment:
I get that there is a different form of "data" for singular and plural, and I find it interesting to know, but if you ask me, the term "datum" is so rarely used nowadays, that it it seems pretty unnatural for Monika to say "datum" when I imagine almost nobody has heard the term, or even known that "data" was not both singular and plural. 

I think the language has evolved in a way where "data" has completely replaced "datum" in everyday usage, and is considered to be both singular and plural. 

This might also make for a good fun fact.

Reply:
after some discussion, we have agreed on changing the line a bit to avoid the data was/were issue entirely.